### ISSUE TITLE
InconsistentLayout Lint Warning

### ISSUE THREAD

**Issue by adhiamboperes** [oppia/oppia-android#5204]
**Cause:** Some of the view items defined in the 'layout-sw600dp' configuration were not found in the main layout resource files.
**Lint warning:** The id "id_name" in layout "resource_file" is missing from the following layout configurations: 
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; layout (present in layout-sw600dp)

**Fix:** Included the absent components in the primary resource layout file.
I have the code changes in my forked repository [Link](https://github.com/Rd4dev/oppia-android/commit/d2d3f2cd44046f28b5974c72641a29609c17153c) for your review and have attached before-and-after screenshots.

**Screenshots**
![Inconsistent layouts oppia-android lint warning before after](https://github.com/oppia/oppia-android/assets/122200035/11cae564-4fc8-4231-a5d0-a5e85304cdd1)

**Comment by adhiamboperes** [comment]
@Rd4dev, PTAL

**Comment by adhiamboperes** [comment]
In https://github.com/Rd4dev/oppia-android/commit/d2d3f2cd44046f28b5974c72641a29609c17153c, have you created the  new views as duplicates of what is in the corresponding files, and have you tested the UI to show that they look good still? Please show before/after screenshots and videos.

**Comment by adhiamboperes** [comment]
@Rd4dev, could you also please fill this out: https://docs.google.com/forms/d/e/1FAIpQLSes4wj7oKtiN0iyUwd-Xt_DAOr1b-MHG9_YYIavWBF4G6g8uA/viewform.

**Comment by Rd4dev** [comment]
Hi @adhiamboperes, Thanks for creating this issue. The UI appeared unchanged but on testing I discovered that certain views are not functioning as intended. I will investigate these issues, make the necessary adjustments, and then proceed to push the pull request.

**Comment by Rd4dev** [comment]
> @Rd4dev, could you also please fill this out: https://docs.google.com/forms/d/e/1FAIpQLSes4wj7oKtiN0iyUwd-Xt_DAOr1b-MHG9_YYIavWBF4G6g8uA/viewform.

@adhiamboperes, Thanks for sharing the collaborator form. The form mentions **"Please do not fill in this form if you haven't merged 2+ PRs yet."**, but I currently have only one merged PR and 2 open PRs. Can I still proceed or wait till I get 2+ PRs merged?

**Comment by adhiamboperes** [comment]
@adhiamboperes, Thanks for sharing the collaborator form. The form mentions **"Please do not fill in this form if you haven't merged 2+ PRs yet."**, but I currently have only one merged PR and 2 open PRs. Can I still proceed or wait till I get 2+ PRs merged?

Please fill it once your secind PR is merged.
